meson.build

```
@@ -0,0 +1,5 @@
+project(
+    'io.elementary.plymouth-theme'
+)
+
+meson.add_postconf_script('meson/postconf.py')
```

That really should be:
```suggestion
meson.add_install_script('meson/postconf.py')
```
With `MESON_INSTALL_DESTDIR_PREFIX` used